vcs/src/main/resources/ext.py

```diff
@@ -167,6 +167,14 @@ def diff_git_raw(ui, repo, rev1, rev2=None, **opts):
         status = repo.status(ctx1)
 
     modified, added, removed = [set(l) for l in status[:3]]
+
+    files = opts['files']
+    if files != '':
+        wanted = set(files.split(','))
```

Comment:
```suggestion
        wanted = set(files.split(b','))
```

This fails on python3